test/models/responses/retrieve/ContactDetailsSpec.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.responses.retrieve
+
+import play.api.libs.json.{JsSuccess, Json}
+import util.SpecBase
+
+class ContactDetailsSpec extends SpecBase {
+
+  "Reads" should {
```

Comment:
You have else where introduce a new after opening a should on tests - I'd either remove those new spaces or add one here so we're consistent.